org.osgi.util.pushstream/src/org/osgi/util/pushstream/package-info.java

```diff
@@ -35,7 +33,7 @@
  * @author $Id$
  */
 
-@Version("1.0.2")
+@Version("1.1.0")
```

Comment:
You also need to update the comment above to use 1.1 instead of 1.0.

Reply:
done